package.json

```diff
@@ -63,7 +63,7 @@
     "jsii-docgen": "^7.2.9",
     "jsii-pacmak": "^1.101.0",
     "jsii-rosetta": "^5",
-    "projen": "^0.84.0",
+    "projen": "^0.84.8",
```

Comment:
Can we update it to newer version? `^0.87.2` as there's recent breaking change to GitHub's upload-artifact action impacted projen managed repositories in cdklabs and was causing our dependency upgrade actions to fail.